app/models/miq_worker.rb

```diff
@@ -345,9 +345,32 @@ def start_runner
     Process.detach(pid)
     pid
   end
+  alias_method :start_runner_via_fork, :start_runner
```

Comment:
I think that I have figured out the "why" for this specific work around, but I still don't like what we are doing.

Instead of the `embedded ansible` worker being a separate process like everything else, it is a `Thread` off the main `MiqServer::Monitor` process.  That seems like it is all sorts of wrong, and doesn't translate to a containerized environment at all.

I can't find any rational as to why we (well... you) made this worker behave so differently, just that it was done (can't even find a PR for it, just a commit:  https://github.com/ManageIQ/manageiq/commit/18ffca9 ).

Regardless of why the worker is like this, the code for this runner seems to be doing complete against a forking/spawning model, meaning it doesn't do either.